test/CMakeLists.txt

```diff
@@ -238,6 +238,7 @@ else ()
           COMMAND root_finders_test -tc=root_finders)
```

Comment:
the indentation in this file seems strange

Reply:
these lines have not been changed